src/components/modebar/buttons.js

```diff
@@ -47,31 +49,31 @@ var modeBarButtons = module.exports = {};
 
 modeBarButtons.toImage = {
     name: 'toImage',
-    title: 'Download plot as a png',
+    title: function(gd) { return _(gd, 'Download plot as a png'); },
```

Comment:
It's kind of annoying to have to turn all these button titles into functions... just to follow the rule that you can only translate a literal string. In principle we could do this translation in `ModeBar.createButton` by adding some complexity to the `find_locale_strings` task to allow one non-literal translation and look for keys inside `modebarButtons`.